src/scaffold/ScaffoldView.tsx

```diff
@@ -118,119 +118,122 @@ const SecondPage: React.FC<PageProps> = ({ onPrevious }) => {
-                      iconUri={bundler.icon}
-                      title={bundler.title}
-                      description={bundler.description}
-                      handleOnClick={() => actions.updateOptions({ bundler: bundler.title })}
-                    />
-                  ))}
+                <div className="bundlersWrapper" css={bundlers}>
+                  <div className="bundlers">
+                    <p className="columnTitle">Select bundler</p>
+                    <div className="bundlersCards" css={cards}>
+                      {state.bundlers.map((bundler) => (
+                        <Card
+                          type="bundler"
+                          key={bundler.title}
+                          iconUri={bundler.icon}
+                          title={bundler.title}
+                          description={bundler.description}
+                          handleOnClick={() => actions.updateOptions({ bundler: bundler.title })}
+                        />
+                      ))}
+                    </div>
+                  </div>
                 </div>
-              </div>
-            </div>
-            <div className="template-specific information" css={infosInputs}>
-              {templateOptions[0] !== 'spinner' &&
-              templateOptions[0] ? (
-                <React.Fragment>
-                  <p className="columnTitle">Provide Template Options</p>
-                  <div className="inputsWrapper">
-                    {state.templateOptions.map((option) => (
-                      <div className="extraItem capitalize" key={option[1].default}>
-                        <VSCodeTextField
-                          className="extraItemInput"
-                          placeholder={option[1].default}
-                          value={option[1].default}
-                          key={option[0]}
-                          onChange={(ev: any) =>
-                            actions.updateOptions({ dynamicOptionValues: [option[0], ev.target.value] })
-                          }>
-                          <p className="extraItemLabel">{option[0]}</p>
-                        </VSCodeTextField>
+                <div className="template-specific information" css={infosInputs}>
+                  {Object.keys(templateOptions).length > 0 && (
+                    <React.Fragment>
+                      <p className="columnTitle">Provide Template Options</p>
+                      <div className="inputsWrapper">
+                        {Object.keys(templateOptions).map((option) => (
+                          <div className="extraItem capitalize" key={templateOptions[option].default}>
+                            <VSCodeTextField
+                              className="extraItemInput"
+                              placeholder={templateOptions[option].default}
+                              value={options.templateOptionsValues[option] || templateOptions[option].default}
```

Comment:
I don't like this very much. For instance, making the field an empty `""` would not be possible here.

A better solution is to pre-populate the `templateOptionsValues` once we have received the `templateOptions`. We should pre-populate these values with the given default values.